api/apps/api/src/modules/published-project/published-project.service.ts

```diff
@@ -64,6 +68,25 @@ export class PublishedProjectService {
       return left(alreadyPublished);
     }
 
+    const scenarioQuery = `SELECT id from scenarios where scenarios.project_id = $1`;
+    const scenarioIds = await this.projectRepository.query(scenarioQuery, [id]);
+    const scenarioIdsForExport =
+      projectToPublish.scenarioIds ||
+      scenarioIds.map((scenario: any) => {
+        return scenario?.id;
+      });
+
+    const exportResult = await this.projectService.requestExport(
```

Comment:
this is good - from the point of view of a user, by now we only know if the request of an export was queued and if we have an `exportId`: if the export will take "a long time" though, initial attempts to clone the exported (or rather, being-exported) project will fail (I think)... do we need to handle this? (not here and not in this PR, but at least in the endpoint to request a clone based on a given `exportId`)

Reply:
Maybe this is what @aciddaute was asking me about some minutes ago? Maybe we need a small meeting to detail the steps because I'm assuming the export works in a specific way that is not exactly what should be covered when publishing?

Reply:
`/api/v1/projects/published-projects/:exportId/clone` endpoint checks wether the export has finished or not